vulnerabilities/templates/packages.html

```diff
@@ -1,64 +1,202 @@
-{% extends 'base.html' %} 
-{% block title %} 
-<h1 class="title">
-Explore Vulnerable packages
-</h1>
+{% extends 'base.html' %}
+{% load humanize %}
+
+{% block title %}
+VCIO Package Search Results
 {% endblock %}
+
 {% block content %}
-{% load widget_tweaks %}
-
-<div class="columns">
-  <div class="column is-one-third is-offset-one-third">
-    <form action="{% url 'package_search' %}" method="get">
-    {% load widget_tweaks %}
-      <div class="field is-horizontal pd-3">
-        <div class="select">
-        {{form.type}}
+
+{% include 'navbar.html' %}
+
+<section class="section01">
```

Comment:
Use descriptive names instead of numbers.

Reply:
Done.